terraform/init/init.tf

```diff
@@ -63,7 +63,7 @@ resource "aws_kms_key" "encrypt" {
 }
 
 resource "aws_kms_alias" "encrypt-alias" {
-  name          = "alias/terraform-state-encryption-key"
+  name          = "${var.prefix}-alias/terraform-state-encryption-key"
```

Comment:
Shouldn't this be `"alias/${var.prefix}-terraform-state-encryption-key"`? I think keys must start with the prefix `alias/`